Source/ActionController.swift

```diff
@@ -253,6 +258,13 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
             }
         }
         
+        // register footer
+        if settings.cancelView.hasCancelView, settings.cancelView.displayInFooter, cancelView != nil {
+            let nib = UINib(nibName: "CancelCell", bundle: Bundle(for: CancelCell.self))
```

Comment:
Here too, you are using CancelCell that is included in the example project, don't you?